src/mongo_conn.js

```diff
@@ -1,37 +0,0 @@
-
-const MongoClient = require('mongodb').MongoClient;
-
-let mongo;
-
-function buildMongoUrl(config) {
-	let hosts = config.hosts;
-	let length = hosts.length;
-	let host = '';
-	for (let i = 0; i < length; i++) {
-		host += hosts[i];
-		if (i < length-1) {
-			host += ',';
-		}
-	}
-	let url = 'mongodb://'
-	if (config.user) {
-		url += config.user + ':'
-	}
-	if (config.password) {
-		url += config.password + '@'
-	}
-	url += host + '/'
-	url += config.database;
-	if (config.options) {
-		url += '?';
-		url += config.options;
-	}
-	return url;

```

Comment:
Hey, @heinrich10 , can you explain why we have multiple hosts here?

Reply:
because when mongodb is on [replica set mode](https://docs.mongodb.com/manual/replication/) the driver would need to know all the ip addresses of the mongodb instance